### DISCUSSION TITLE
Change event open menu of modal 3D

### DISCUSSION THREAD

**Discussion by nhatdn** [mozilla/hubs#5756] [General Questions & Discussion] [answered]
i want to change event open menu for modal 3D but i dont know it. Please helpe me.. thanks all
![image](https://user-images.githubusercontent.com/85542339/197328806-3a4c5dd7-87e6-4934-b602-70785fb60182.png)

**Comment by matthewbcool** [comment, marked as answer]
To edit the in-room UI, you would need to run a custom client. https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html

Welcome to join the developer community in our discord for more tips and to ask specific questions related to the code. https://discord.gg/Jer5t54U

**Reply by nhatdn** [reply]
i cloned hub repository and run it at my local.. i only want to ask where are code, because i want to custom menu but i dont know that code anywhere, please help me if you can.
